domains/primitives/beacon/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
```

Comment:
i think the crate name should be more than just `beacon`. Just beacon is not clear what is it for